ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-oadp-1.1__4.9.yaml

```
@@ -0,0 +1,120 @@
+base_images:
+  base:
+    name: "4.9"
+    namespace: ocp
+    tag: base
+  openshift_release_golang-1.17:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.17-openshift-4.10
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.17
+images:
+- dockerfile_path: Dockerfile
+  from: base
+  to: oadp-operator
+- dockerfile_path: build/ci-Dockerfile
+  from: openshift_release_golang-1.17
+  to: test-oadp-operator
+operator:
+  bundles:
+  - dockerfile_path: build/Dockerfile.bundle
+  substitutions:
+  - pullspec: quay.io/konveyor/oadp-operator:latest
+    with: oadp-operator
+releases:
+  latest:
+    release:
+      channel: stable
+      version: "4.9"
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 1000m
+      memory: 512Mi
+tests:
+- as: operator-e2e-aws-periodic-slack
+  cron: 13 22 */13 * *
+  steps:
+    cluster_profile: aws
+    dependencies:
+      OO_INDEX: ci-index
+    env:
+      OO_CHANNEL: stable
+      OO_INSTALL_NAMESPACE: openshift-adp
+      OO_PACKAGE: oadp-operator
+      OO_TARGET_NAMESPACES: '!install'
+    test:
+    - as: e2e
+      cli: latest
+      commands: CLUSTER_PROFILE=aws make test-e2e
+      credentials:
+      - mount_path: /var/run/oadp-credentials
+        name: oadp-credentials
+        namespace: test-credentials
+      from: test-oadp-operator
+      resources:
+        requests:
+          cpu: 1000m
+          memory: 512Mi
+    workflow: optional-operators-ci-aws
+- as: operator-e2e-gcp-periodic-slack
+  cron: 22 21 */14 * *
+  steps:
+    cluster_profile: gcp
+    dependencies:
+      OO_INDEX: ci-index
+    env:
+      OO_CHANNEL: stable
```

Per https://github.com/openshift/oadp-operator/blob/oadp-1.1/bundle/metadata/annotations.yaml#L8 should now be the following
```suggestion
      OO_CHANNEL: stable-1.1
```